connect/runtime/src/main/java/org/apache/kafka/connect/runtime/standalone/StandaloneHerder.java

```diff
@@ -110,7 +110,7 @@ private ConnectorInfo createConnectorInfo(String connector) {
         if (!configState.contains(connector))
             return null;
         Map<String, String> config = configState.connectorConfig(connector);
-        return new ConnectorInfo(connector, config, configState.tasks(connector));
+        return new ConnectorInfo(connector, config, configState.tasks(connector), connectorType(connector));
```

Comment:
The `connector` parameter is actually the _connector name_, not the connector class name. And as mentioned above, the `connectorType(String)` method expects the class name. So this needs to be addressed, likely by getting the classname from the configuration. The fact that this is being done in at least 3 places seems like something can be made simpler.